addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18633,7 +18633,7 @@ msgstr ""
 #. Description of setting "System -> Video output -> 3D LUT" with label #36564
 #: system/settings/settings.xml
 msgctxt "#36565"
-msgid "TODO: 3D LUT filename description"
+msgid "Select a 3DLUT file to be used by default. If you place multiple 3DLUT files in the same directory, you can switch between them in the OSD menu during video playback. This allows for multiple display profiles to account for different viewing environments and source materials. A few ideas as starting points: gamma 2.2 for daytime viewing, 2.4 for nighttime; BT.601 primaries for SD material or older HD, BT.709 for modern HD; gamma input offset to avoid black crushing, output offset or absolute gamma for material with grey blacks."
 msgstr ""
```

Comment:
I would change it to:

"Select **the** 3DLUT file to be used by default. If you place multiple 3DLUT files in the same directory**, then** you can switch between them in the OSD menu during video playback. This allows for multiple display profiles to account for different viewing environments and source materials. For example profiles for gamma 2.2 for daytime viewing, and 2.4 for night time."

A couple of nitpicks and no need for so many examples in my view, the wiki is the correct place for that.